frontend/src/components/Home.js

```diff
@@ -1,26 +1,15 @@
 import React, { useState, useEffect } from 'react';
-
 import { useHistory } from 'react-router';
-import {
-  makeStyles,
-  Button,
-  Box,
-  TableContainer,
-  Table,
-  TableHead,
-  TableBody,
-  TableRow,
-  TableCell,
-} from '@material-ui/core';
+import { makeStyles, Button, Box } from '@material-ui/core';
 import EditOutlinedIcon from '@material-ui/icons/EditOutlined';
 import DeleteOutlineOutlinedIcon from '@material-ui/icons/DeleteOutlineOutlined';
```

Comment:
These imports are also unused after the refactor.

Reply:
I'll take them out!